src/3d/qgspointcloudlayer3drenderer.h

```diff
@@ -27,9 +27,73 @@
 
 class QgsPointCloudLayer;
 #include "qgspointcloud3dsymbol.h"
+#include "qgsfeature3dhandler_p.h"
 
 #ifndef SIP_RUN
 
+/**
+ * \ingroup core
+ * \class QgsPointCloud3DRenderContext
+ *
+ * Encapsulates the render context for a 3D point cloud rendering operation.
+ *
+ * \since QGIS 3.18
+ */
+class CORE_EXPORT QgsPointCloud3DRenderContext : public Qgs3DRenderContext
+{
+  public:
+
+    /**
+     * Constructor for QgsPointCloud3DRenderContext.
+     *
+     * The \a scale and \a offset arguments specify the scale and offset of the layer's int32 coordinates
+     * compared to CRS coordinates respectively.
```

Comment:
I can't see scale/offset arguments in the constructor :-)

Reply:
Leftover from QgsPointCloud2DRenderContext